index.bs

```diff
@@ -7814,16 +7816,26 @@ To <dfn export>run WebDriver BiDi preload scripts</dfn> given |environment setti
 
 1. Let |navigable| be |document|'s [=/navigable=].
 
+1. Let |user context| be |navigable|'s [=associated user context=].
+
+1. Let |user context id| be |user context|'s [=user context id=].
+
 1. For each |session| in [=active BiDi sessions=]:
 
   1. For each |preload script| in |session|'s [=preload script map=]'s
      [=values=]:
 
+    1. If |preload script|'s <code>user contexts</code>'s [=set/size=] is not zero:
+
+       1. If |preload script|'s <code>user contexts</code> does not [=list/contain=] |user context id|,
```

Comment:
So this is giving us "and" semantics for preload scripts i.e. you have to match both a user context id and a context id, if both are provided. For subscriptions, I think we have "or" semantics i.e. matching one or the other is fine. Generally I think I'd prefer either "or" semantics or we only allow the commands to provide either user contexts or contexts, but not both.

Reply:
Indeed, changed to the "or" semantics, I think the "and" logic does not make sense here, same as in events.

Reply:
Should we make `contexts` and `userContexts` mutually exclusive?

Reply:
This would simplify understanding API edge cases.

Reply:
They are mutually exclusive in the latest patch.

Reply:
UPD: oh, they already are, so nevermind.